cli/src/pcluster/cli.py

```diff
@@ -331,68 +336,21 @@ def _get_parser():
 
     # createami command subparser
     pami = subparsers.add_parser(
-        "createami", help="(Linux/macOS) Creates a custom AMI to use with AWS ParallelCluster."
-    )
-    pami.add_argument(
-        "-ai",
-        "--ami-id",
-        dest="base_ami_id",
-        required=True,
-        help="Specifies the base AMI to use for building the AWS ParallelCluster AMI.",
-    )
-    pami.add_argument(
-        "-os",
-        "--os",
-        dest="base_ami_os",
-        required=True,
-        help="Specifies the OS of the base AMI. "
-        "Valid options are: alinux, ubuntu1604, ubuntu1804, centos7, centos8.",
+        "build-image", help="(Linux/macOS) Creates a custom AMI to use with AWS ParallelCluster."
```

Comment:
I think now it can work for Windows too, because we removed the execution of the bash script.

Reply:
sorry, I miss something, which bash script you mentioned here?

Reply:
I mean, the previous create-ami command didn't work on windows because it executed the build-ami.sh, the new build-ami can work on windows, so we can remove the `(Linux/macOS)` from the help.